src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/Msg.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg;
+
+import java.util.UUID;
+
+public abstract class Msg {
```

Comment:
why is this abstract this looks like a POJO class to me ?

Reply:
Because it's our basic message for the service bus queues that will have specific implementations for envelope messages or other. This is not really the final version, as I am making changes to this, but this will probably stay abstract (there will be "stuff" added to it though).

Reply:
...actually it might become an interface...but you will only know when I am done ;-) (I'll also remove the WIP then).

Reply:
I've opted for an interface in the end (for the time being at least). This is used to represent messages as seen by the application which can be easily mapped to azure servicebus messages.